### PR TITLE
Add padding to build log terminal

### LINKED ISSUE #2534: Add padding to build log terminal
The build log terminal's content is pressed right against the edge, and doesn't look as nice as it could.  Let's add a small bit of padding so there is some extra "black" space.

Screen Shot 2021-11-28 at 10 46 05 AM

### PR DESCRIPTION
**Pull request by menghif**
## Issue This PR Addresses

Fixes #2534 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

Add `1rem` padding to build log terminal.

Screen Shot 2021-11-29 at 10 27 27 AM

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/api/status/public/css/terminal.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#terminal .xterm-cursor-layer {
+  padding: 1rem;
```
Comment (humphd) [inline src/api/status/public/css/terminal.css:2]: I think it's too much, `.5rem`?

**Review by humphd — CHANGES_REQUESTED** [review]

**Review by humphd — DISMISSED** [review]

**Review by Andrewnt219 — CHANGES_REQUESTED** [review]
Seem like it isn't working. Reproduce:

```diff
// terminal.js
+ terminal.write("Hello world");
```

image

**Review by Yoda-Canada — COMMENTED** [review]
@menghif I retest your code, and the cursor can display on the second line. However when I put in some letters, they are still on the first line. 
image
image
The **convertEol: true,** means "When enabled the cursor will be set to the beginning of the next line with every new line. This is equivalent to sending '\r\n' for each '\n'. " Therefore, my point is that **convertEol**  is the issue.
```
const terminal = new Terminal({
  cols: 80,
  rows: 30,
  convertEol: true,
  theme,
});
```

**Comment by humphd** [comment]
The fix for this is to use CSS to wrap the terminal in a parent element, then set its background to the same colour and include padding.  Here's an example of how I do this on another project:

Screen Shot 2021-12-02 at 11 08 23 AM

**Comment by Andrewnt219** [comment]
@menghif Add padding to the class ".xterm" works as well. See https://github.com/xtermjs/xterm.js/pull/1208

image

**Comment by humphd** [comment]
Can we get an update on this?

**Comment by menghif** [comment]
Sorry for the delay, @Andrewnt219's fix seems to work well!

**Review by Andrewnt219 — DISMISSED** [review]
LGTM 👍.

**Review by humphd — DISMISSED** [review]

**Comment by humphd** [comment]
@menghif squash and rebase this, and I'll merge it!

**Review by Andrewnt219 — APPROVED** [review]

**Comment by humphd** [comment]
fyi @Yoda-Canada, this might help with your work on styling the terminal as well.

**Pull request merged**